extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -248,9 +247,9 @@ private function clearUsers() {
 	 * @return string homepage URL. Default is US homepage.
 	 */
 	private function getHomepageUrl() {
-		global $wgLang;
+		global $wgContLang;
 		if ( class_exists('WikiaLogoHelper') ) {
-			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgLang->getCode() );
+			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgContLang->getCode() );
```

Comment:
you sure about this change? It changes behavior a little bit.
Before: If user was logged in he would be redirected wikia in his lang
Now: It doesn't matter if user is logged in or not he always will be redirected to wikia in content lang.

I'm not sure if it works that way on desktop and I'm not sure if we want to be consistent with desktop but I would like to be sure we are aware of this change and it has some purpose